packages/aml-checklist/package.json

```diff
@@ -22,6 +22,8 @@
     "lint": "concurrently \"tsc --noEmit\" \"eslint --cache --ext=ts,tsx,js src\""
   },
   "dependencies": {
+    "@reapit/elements": "^0.5.39",
```

Comment:
why do we need to add this ?

Reply:
We need this so `yarn build` know which package to build first.
For example:
`aml-checklist` depends on `@reapit/elements` & `@reapit/cognito-auth`. and `@reapit/cognito-auth` depends on `@@reapit/elements`. Then `lerna/yarn workspace` will build `@reapit/elements` first -> then `@reapit/cognito-auth` -> `aml-checklist`

Reply:
Example from material-ui: https://github.com/mui-org/material-ui/blob/master/packages/material-ui/package.json#L49

Reply:
@vuhuucuong could we move this to root elements instead of let it on each packages ?

Reply:
I think no, because if we move it to root `package.json`, then `lerna/yarn workspace` cannot know which package depends on which package -> then `yarn build` will build in the wrong order -> and cause our pipeline to fail again
For now, I'm finding a way to config our `webpack/tsconfig` so  that we don't need to set paths alias in tsconfig.json in each package anymore or use one `tsconfig.json` for alias all packages like this
https://github.com/mui-org/material-ui/blob/master/tsconfig.json